src/resolvers.ts

```diff
@@ -30,6 +30,7 @@ export type InjectorFunction = <T extends object>(
  */
 export interface Resolver<T> extends ResolverOptions<T> {
   resolve<U extends object>(container: AwilixContainer<U>): T
+  isValue: boolean
```

Comment:
`Resolver` is a public interface and is used by consumers to implement their own resolvers. Adding a required property here would be a breaking change.

Reply:
good point. I'll figure out a way to do this that doesn't change the public interfaces.

Reply:
We can add stuff as long as they are optional.